shogun-lib/src/main/java/de/terrestris/shogun/lib/security/access/entity/BaseEntityPermissionEvaluator.java

```diff
@@ -316,14 +403,32 @@ private boolean containsReadPermission(ClassPermission ...classPermissions) {
     }
 
     protected boolean hasPublicPermission(E entity) {
-
         if (entity instanceof Group || entity instanceof User) {
             return false;
         }
 
         return publicInstancePermissionService.getPublic(entity);
     }
 
+//    /**
+//     * Returns the roles of the currently authenticated user.
+//     * @return
+//     */
+//    protected List<Role> getUserRoles() {
+//        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+//
+//        Collection<? extends GrantedAuthority> grantedAuthorities = authentication.getAuthorities();
+//
+//        return grantedAuthorities.stream()
+//            .map(grantedAuthority -> {
+//                String authority = grantedAuthority.getAuthority();
+//                return roleRepository.findByName(authority);
+//            })
+//            .filter(Optional::isPresent)
+//            .map(Optional::get)
+//            .toList();
+//    }
+
```

Comment:
Can this be removed, if not, does it belong to this class or should this moved somewhere else? Like the `RoleService` or `UserService`?

Reply:
Can be removed (see #8179ef7)